packages/web/components/Player/PlayerTimeZone.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, HStack, Text } from '@metafam/ds';
+import { PlayerFragmentFragment } from 'graphql/autogen/types';
+import React from 'react';
+import { FaGlobe } from 'react-icons/fa';
+import spacetime from 'spacetime';
+import { display } from 'spacetime-informal';
+
+type Props = {
+  player: PlayerFragmentFragment;
+};
+
+export const PlayerTimeZone: React.FC<Props> = ({ player }) => {
+  let tzLabel = '-';
+  let offsetLabel = null;
+  if (player.tz) {
+    const a = spacetime.now().goto(player.tz)
```

Comment:
what is "a" supposed to be? A more descriptive name would make it more readable.